tools/myst/host/syscall.c

```diff
@@ -520,3 +520,14 @@ long myst_fsync_ocall(int fd)
 {
     RETURN(fsync(fd));
 }
+
+long myst_get_mempolicy_ocall(
+    int* mode,
+    unsigned long* nodemask,
+    size_t nodemask_size,
+    unsigned long maxnode,
+    void* addr,
+    unsigned long flags)
+{
+    RETURN(syscall(SYS_get_mempolicy, mode, nodemask, maxnode, addr, flags));
```

Comment:
OE outcall implementation does NOT zero-set [OUT] parameter. Does Linux syscall implementation zero out nodemask before setting relevant bits? If not, the returned nodemask might be a mix of SYSCALL set bits and the relic of nodemask allocated by the OE OCALL infrastructure code. Setting nodemask as [IN, OUT] in the EDL will guarantee that nodemask is zeroed before the Linux syscall.